### PR TITLE
Updating vm version on the cmake

### PR DESCRIPTION
**Pull request by jordanmontt**
This PR changes the vm version of the `vmmaker.cmake` file. I took the latest releases of [https://files.pharo.org/vm](https://files.pharo.org/vm).

I did not update the vm of `pharo-spur64-headless/Linux-aarch64` and `pharo-spur32-headless/Linux-armv7l` because there was not new versions released for those OSs.

~One thing, I didn't change the hash, which I suppose that also changed. Where are the hashes available? I didn't find them~

### CODE REVIEW

**Comment by guillep** [comment]
Thanks @jordanmontt 

But we should update the image together with the VM.

**Comment by jordanmontt** [comment]
Done! The CI is still failing however :/

File: cmake/vmmaker.cmake [hunk 1]
```diff
@@ -141,8 +141,8 @@ if(GENERATE_SOURCES)
         ExternalProject_Add(
             vmmaker
 
-            URL https://files.pharo.org/image/110/Pharo11-SNAPSHOT.build.688.sha.cf3d3fd.arch.64bit.zip
-            URL_HASH SHA256=c050ddcedce70ec92c22a3244aa5ebbc655dcaffcb42ac80fbf1f6e795c7010d
+            URL https://files.pharo.org/image/110/Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip
+            URL_HASH SHA256=b96a943513c6a79c7319b285e46719fbe74c570ace3fa0c9ead43123c165671a
             BUILD_COMMAND ${VMMAKER_VM} --headless ${VMMAKER_DIR}/image/Pharo11-SNAPSHOT-64bit-cf3d3fd.image --no-default-preferences save VMMaker
```
Comment (guillep) [inline cmake/vmmaker.cmake]: Now it is failing because the hash in this line is not right and it does not find the image :)

Comment (jordanmontt) [reply cmake/vmmaker.cmake]: I re-ran ` sha256sum Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip` and it gives me that hash that is committed. And the image `Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip` is the last one built on files.pharo.org

Comment (guillep) [reply cmake/vmmaker.cmake]: I know, I selected the line below ;)

Comment (jordanmontt) [reply cmake/vmmaker.cmake]: 🤦Dumb Sebas

**Comment by guillep** [comment]
@jordanmontt if you don't mind I'll redo this PR using the P12 release ;)

**Pull request closed without merging**